Tests/Unit/UseCases/LoginUseCaseTest.php

```diff
@@ -53,21 +53,26 @@ public function testPasswordNotMatch()
     {
         
         
-        $this->UserRepository
+        $this->userRepository
             ->expects($this->once())
             ->method('findOneBy')
             ->with(
                 ['username' => 'username']
             )
-            ->willReturn($this->user);
+            ->willReturn('username');
```

Comment:
Y esto lo tienes que dejar como estaba antes. Si te fijas el método `findOneBy` nos devuelve un objeto de tipo usuario, tal y como está ahora estamos diciendo que devuelve el string `username`, por lo tanto no es correcto. Lo que nos devolverá será `$this->user`